docs/install/install-redis/install-redis-on-linux.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: "Install Redis on Linux"
+title: "Install Valkey on Linux"
 linkTitle: "Linux"
 weight: 1
 description: >
-    How to install Redis on Linux
+    How to install Valkey on Linux
 aliases:
 - /docs/getting-started/installation/install-redis-on-linux
 ---
 
-Most major Linux distributions provide packages for Redis.
+Most major Linux distributions provide packages for Valkey.
 
 ## Install on Ubuntu/Debian
```

Comment:
This is not correct as of today. We don't have a debian based package yet.

Reply:
Shall we remove it?

I was thinking we can also remove it after bulk-rebranding.

Reply:
Yeah, I somewhat expect that most of this content will change.

Reply:
Deleted.